packages/now-cli/src/util/dev/server.ts

```diff
@@ -1646,15 +1648,12 @@ export default class DevServer {
       ...process.env,
       ...this.buildEnv,
       NOW_REGION: 'dev1',
+      PORT: `${port}`,
     };
 
-    const devCommand = this.devCommand
-      .replace(/\$PORT/g, `${port}`)
-      .replace(/%PORT%/g, `${port}`);
```

Comment:
We need to keep this, because `$PORT` does not work on Windows.

Reply:
Thanks, I fixed it 🔧